tt_metal/hw/inc/risc_common.h

```diff
@@ -211,4 +211,25 @@ inline void breakpoint_(uint32_t line) {
 
 #define breakpoint() breakpoint_(__LINE__);
 
+// Helper function to wait for a specified number of cycles, safe to call in erisc kernels.
+#if defined(COMPILE_FOR_ERISC)
+// Forward declaration to avoid circular dependency
+namespace internal_ {
+void __attribute__((section("code_l1"))) risc_context_switch();
+void disable_erisc_app();
+}
+#endif
+inline void riscv_wait(uint32_t cycles) {
```

Comment:
This is a modified version of the wait() function from ckernels.h, which was moved into the llk repo.